src/components/dialogs/AccountDialogForm.vue

```diff
@@ -22,6 +22,17 @@
               <el-option v-for="item in statuses" :key="item.value" :label="item.label" :value="item.value" />
             </el-select>
           </el-form-item>
+          <el-form-item
+            :label="$t('accounts-view.form.banner-message.text')"
+            prop="bannerMessageText"
+          >
+            <el-input v-model="form.bannerMessage.text" @blur="setTimestampBannerMessage" />
+            <el-switch
+              v-model="form.bannerMessage.isVisible"
+              :inactive-text="$t('accounts-view.form.banner-message.switch-inactive-message')"
+              :active-text="$t('accounts-view.form.banner-message.switch-active-message')"
+            />
```

Comment:
Niveau UI, je pensais plus à aligner le switch avec le label de l'input donc on n'aurait plus besoin des trads.

J'imaginais ça :
```suggestion
            <el-switch v-model="form.bannerMessage.isVisible" />
            <el-input
              v-model="form.bannerMessage.text"
              :disabled="!form.bannerMessage.isVisible"
              @blur="setTimestampBannerMessage"
            />
```